components/layout_thread/lib.rs

```diff
@@ -954,6 +954,21 @@ impl LayoutThread {
             }
             let display_list = (*rw_data.display_list.as_ref().unwrap()).clone();
 
+            for item in &display_list.list {
+                match *item {
+                    DisplayItem::Iframe(ref iframe) => {
+                        let size = Size2D::new((*item).bounds().size.width.to_f32_px(),
+                                               (*item).bounds().size.height.to_f32_px());
+                        debug!("new iframe size: {:?}", size);
```

Comment:
In fact, we may want to consider batching up all of the iframe sizes that are discovered and sending them in a single message once the display list has been created, rather than one by one.